### PR TITLE
Add supporters section to landing page

### LINKED ISSUE #25: Add `Who funds SV2 development section`
Current supporters supporting devs are known and can be added.

- [https://Spiral.xyz](https://www.figma.com/exit?url=https%3A%2F%2Fspiral.xyz)
- [https://www.galaxy.com](https://www.figma.com/exit?url=https%3A%2F%2Fwww.galaxy.com)
- [http://foundrydigital.com](https://www.figma.com/exit?url=http%3A%2F%2Ffoundrydigital.com)
- [http://summerofbitcoin.org](https://www.figma.com/exit?url=http%3A%2F%2Fsummerofbitcoin.org)

Screenshot 2022-08-31 at 23 33 27

https://www.figma.com/file/H42PrGnniKFtbfqHYBGevj/Stratum?node-id=0%3A1

### PR DESCRIPTION
**Pull request by secondl1ght**
Fixes #25 

Added this section to match design, I also created a new `PrimaryLink` component for the green button because we will want to use it in a couple different places. We can build out this component more as needed unless you had a different idea for this @SeraBocca?

Screenshots:

image
image

And then they stack on mobile. 🚀

### CODE REVIEW

File: src/.vuepress/theme/layouts/HomeLayout.vue [hunk 1]
```diff
@@ -89,11 +89,11 @@
       ></div>
     </div>
 
-    <div class="flex flex-col pt-36 mb-10 z-[20]">
+    <div class="flex flex-col pt-36 z-[20]">
       <div class="flex flex-col mb-44">
         <h1 class="mb-8 text-3xl lg:text-5xl">{{ data.introductionTitle }}</h1>
         <p
-          class="w-[1040px] max-w-[90vw] text-center mt-O mb-12 mx-auto text-base lg:text-2xl"
```
Comment (SeraBocca) [inline src/.vuepress/theme/layouts/HomeLayout.vue:96]: I put this breakpoint to better match small device width, we have a use case that requires removing it?

File: src/.vuepress/theme/components/PrimaryLink.vue [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<template>
```
Comment (SeraBocca) [inline src/.vuepress/theme/components/PrimaryLink.vue:1]: Good idea! i will use it in this [PR](https://github.com/stratum-mining/stratumprotocol.org/pull/29) and improve it to handle internal links

File: src/.vuepress/theme/layouts/HomeLayout.vue (hunk 1 above)
Comment (secondl1ght) [reply src/.vuepress/theme/layouts/HomeLayout.vue:96]: I just thought it looked a bit better this way but we can switch it back. I dont think we have mobile designs for the whole page, we should also make sure the fonts are consistent across the different sections too.

**Comment by secondl1ght** [comment]
Closing due to GH bug - new PR with resolved conflicts is #33

**Pull request closed without merging**